west.yml

```diff
@@ -54,7 +54,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 35654ffa6d06c49912e1af93cf6548895624e6dc
+      revision: pull/656/head
```

Comment:
Please update SHA: `f80aea343f37aa4d5e3fb707ce2dcf842bf0e06a`

Reply:
Done.

Reply:
Rebased/resolved a conflict. Hopefully CI is ok after that.